.github/workflows/ci.yml

```diff
@@ -18,21 +18,16 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 1
     - run: |
         git fetch --prune --unshallow --tags
         echo exit code $?
         git tag --list
-    - uses: actions/setup-dotnet@v1
+    - uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: | 
-          5.0.x
-          6.0.x
-        source-url: https://nuget.pkg.github.com/elastic/index.json
-      env:
-        NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        dotnet-version: 8.0.x
```

Comment:
Just a suggestion: I have good experience using the "global-json-file" input instead of manually specifying the version here. That saves you the work of manually keeping the SDK versions in sync.

Reply:
Yeah, that's a good suggestion. Thanks!